src/patroni.py

```diff
@@ -306,13 +306,30 @@ def member_started(self) -> bool:
             allow server time to start up.
         """
         try:
-            for attempt in Retrying(stop=stop_after_delay(90), wait=wait_fixed(3)):
+            for attempt in Retrying(stop=stop_after_delay(30), wait=wait_fixed(3)):
```

Comment:
This whole change to the `member_started` check is more a suggestion, I think it could be useful to check with inverted `TLS_enabled` flag in case it fails (also think this improves stability). Apparently this change made codecov sad because we don't have test coverage for this function? LMK what you think about it

Reply:
Did you track down the specific situation where the inverted `TLS_enabled` flag was needed? I mean, in a very specific situation, we are probably setting the wrong value to `self._tls_enabled`. IMO, that information is really important.

Reply:
Maybe we can also check if the cert files exist in `self._tls_enabled`? I can't think of another reason for it to be incorrect.

Reply:
After further testing, I decided to revert this specific change. During my initial testing, I was led to believe that this extra check was improving test stability, but now after re-analysis I don't see any clear indication that it improved anything. Therefore, for the time being, I think its better to not include it.

I'll keep an eye out to see if any issues arise due to this check failing because of wrong `tls_enabled`, but that will be for outside this PR. Thanks for the discussion!